pennylane/templates/subroutines/qsvt.py

```diff
@@ -953,42 +953,43 @@ def circuit_qsvt():
 
     """
 
+    poly_list = list(poly)
```

Comment:
Why do we need `list` here?

Reply:
To eliminate unnecessary trailing zeros in a simple way.
Discussion in [slack](https://xanaduhq.slack.com/archives/C0743CNS9E3/p1739978831180799?thread_ts=1739966207.184529&cid=C0743CNS9E3) for reference